CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 9.0.0 (TBD)
```

Comment:
Remove this. I have an open PR with a line for these changes, I'll link to your PR when it's merged in.

Reply:
Actually I was relying with this change on what you did in #2341. I think in general this kind of thing should be done on release branch(?) ➡️ Reverted